pkgs/development/python-modules/ssort/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, fetchFromGitHub, buildPythonPackage, setuptools, pathspec, }:
+
+buildPythonPackage rec {
+  pname = "ssort";
+  version = "0.13.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "bwhmather";
+    repo = "ssort";
+    rev = "refs/tags/${version}";
+    hash = "sha256-P/FUayCC7KfXjtzclTPLhLw5o0bV4L98tes69w+038o=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ pathspec ];
+
+  pythonImportsCheck = [ "ssort" ];
+
+  # no check
+  doCheck = false;
+
+  meta = {
+    description = "Automatically sorting python statements within a module";
+    homepage = "https://github.com/bwhmather/ssort";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ tochiaha ];
+    mainProgram = "ssort";
+    platforms = lib.platforms.all;
```

Comment:
Iirc you dont need to set `platforms` for python modules, may a other maintainer confirm this
```suggestion
```